File: 0001 Code Chef/0026 How many unattempted Problem/implementation/0009 dart/unattempted.dart
import 'dart:io';

void main(){
  String? stringInput = stdin.readLineSync();
  if(stringInput != null){
    List<String>? stringArray = stringInput.split(" ");
    if(stringArray != null && stringArray.length == 2){
      int? total = int.tryParse(stringArray[0]);
      int? attempted = int.tryParse(stringArray[1]);
      if(total != null && attempted != null){
        int unattempted = total - attempted;
        print(unattempted);
      }
    }
  }
}

File: 0001 Code Chef/0026 How many unattempted Problem/implementation/0010 nodejs/unattempted.js
const readline = require('readline');

const rl = readline.createInterface({
    input:process.stdin,
    output:process.stdout
});

rl.question('',(input)=>{
    let stringArray = input.split(" ");
    let total = parseInt(stringArray[0]);
    let attempted = parseInt(stringArray[1]);
    let unattempted = total - attempted;
    console.log(unattempted);
    rl.close();
});